content/docs/introducing-jsx.md

```diff
@@ -6,31 +6,37 @@ prev: hello-world.html
+Dit heet JSX en is een syntax uitbreiding voor JavaScript.
+Wij raden aan om het samen met React te gebruiken om te beschrijven hoe de UI er uit moet zien.
+JSX doet je misschien denken aan een template taal, maar het heeft alle voordelen van JavaScript.
 
-JSX produces React "elements". We will explore rendering them to the DOM in the [next section](/docs/rendering-elements.html). Below, you can find the basics of JSX necessary to get you started.
+JSX produceert React "elementen". 
+In het [volgende hoofdstuk](/docs/rendering-elements.html) gaan we bekijken hoe deze in de DOM gerenderd worden.
+Hieronder vind je de basis van JSX die je nodig hebt om te beginnen.
 
-### Why JSX? {#why-jsx}
+### Waarom JSX? {#why-jsx}
 
-React embraces the fact that rendering logic is inherently coupled with other UI logic: how events are handled, how the state changes over time, and how the data is prepared for display.
+React omarmt het feit dat de logica voor het renderen vanzelfsprekend gekoppeld is met andere UI logica: hoe events worden afgehandeld, hoe de state voortdurend verandert en hoe de data wordt voorbereid voor weergave.
 
-Instead of artificially separating *technologies* by putting markup and logic in separate files, React [separates *concerns*](https://en.wikipedia.org/wiki/Separation_of_concerns) with loosely coupled units called "components" that contain both. We will come back to components in a [further section](/docs/components-and-props.html), but if you're not yet comfortable putting markup in JS, [this talk](https://www.youtube.com/watch?v=x7cQ3mrcKaY) might convince you otherwise.
+In plaats van kunstmatig de *technologieën* te scheiden door opmaak en logica in verschillende bestanden te plaatsen, gebruikt React los gekoppelde eenheden, genaamd "componenten", welke beiden bevat om de [*verantwoordelijkheden* te scheiden (Engels)](https://nl.wikipedia.org/wiki/Separation_of_concerns). 
+In een [latere sectie](/docs/components-and-props.html) komen we terug op componenten, maar als je het nog ongemakkelijk vindt om opmaak in JavaScript te schrijven, dan kan [deze presentatie (Engels)](https://www.youtube.com/watch?v=x7cQ3mrcKaY) je misschien nog overtuigen.
 
-React [doesn't require](/docs/react-without-jsx.html) using JSX, but most people find it helpful as a visual aid when working with UI inside the JavaScript code. It also allows React to show more useful error and warning messages.
+Het is in React [niet verplicht](/docs/react-without-jsx.html) om JSX te gebruiken, maar de meesten vinden het handig als visueel hulpmiddel bij het werken aan de UI in JavaScript code.
+Tevens zorgt het voor betere foutmeldingen en waarschuwingen.
 
-With that out of the way, let's get started!
+Nu we dat gehad hebben, is het tijd om te beginnen!
```

Comment:
```suggestion
Nu we dat achter de rug hebben, is het tijd om te beginnen!
```
Sounds slightly better to me. Might be a Flemish-Dutch thing though, so definitely decide for yourself.

Reply:
No, i like your suggestion!